authors/apps/authentication/serializers.py

```diff
@@ -143,3 +145,51 @@ def update(self, instance, validated_data):
         instance.save()
 
         return instance
+
+
+class ForgotPasswordSerializer(serializers.ModelSerializer):
+    """Performs email serialization."""
+    email = serializers.EmailField()
+
+
+class ResetPasswordSerializers(serializers.ModelSerializer):
+    """
+    Performs reset password fields serialization.
+    """
+    email = serializers.EmailField(max_length=255)
+    password = serializers.CharField(
+        max_length=128,
+        min_length=8
+    )
+    confirm_password = serializers.CharField(
+        max_length=128,
+        min_length=8
+    )
+
+    token = serializers.CharField(max_length=255)
+
+    def validate(self, data):
+        """
+        Validates passwords match and
+        token can only be used once
+        """
+
+        # Query user from DB using email
+        email = data.get('email')
+        user = User.objects.filter(email).first()
+
+        if data.get('password') != data.get('confirm_password'):
+            msg = "Passwords do not match"
+            raise serializers.ValidationError(msg)
+
+        # Confirm if token is valid
+        token = data.get('token')
+        validate_token = default_token_generator.check_token(user, token)
```

Comment:
Since we are using a custom token generator, this PR https://github.com/andela/ah-code-blooded/pull/11 requires to be merged first and then create a `token` validation method in the backends. 
Check this question on SO. https://stackoverflow.com/questions/46234627/how-does-default-token-generator-store-tokens